corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -83,6 +82,9 @@ hqDefine("cloudcare/js/formplayer/menus/collections", function () {
                 _.extend(this, _.pick(response, this.detailProperties));
                 return response.details;
             } else if (response.tree) {
+                if (response.selections) {
```

Comment:
This clears selections on form entry, right? That sounds good, but has some potential edge cases:

* If the user never goes into the form - they click back to the home screen, or an earlier breadcrumb - will selections get cleared?
* If the multi select case list is the first of two case lists (parent child workflow), the selections will still be present for the second case list - is that okay? (Do we even support this workflow, multi select parent and single select child? I'm pretty sure we don't support two multi select case lists.)